### ISSUE TITLE
Tag Sources from Discogs throwing an error

### ISSUE THREAD

**Issue by Kevlet** [puddletag/puddletag#796]
### Current Behavior

Error message shown whenever Tagging Sources using Source Discogs.com

### Expected Behavior

Tags from Discogs should be shown

### Steps To Reproduce

1. Tag Sources
2. Select Discogs.coma
3. Enter release ID e.g.  26744849 and click "Search"
4. No tags are shown and message: "An error occurred: local variable "__reference_time" referenced before assignment

### Information about your system

Puddletag: 2.2.0
OS: Linux Mint 20
Python: 2.7.18

**Comment by Kevlet** [comment]
Typo:
Select Discogs.**com**

**Comment by bernd-wechner** [comment]
Heads up that I'm happy top look at this as soon this https://github.com/puddletag/puddletag/pull/694 is merged and I find time. I'm waiting on that to continue working discogs (I have a branch in progress as I want to use it for cover art more effectively). That said, if you're in a hurry, puddletag is in Python and eminently hackable and diagnosable, and this looks like a dead easy bug to trace and fix, iff (i.e. if and only if) it's reproducible.

**Comment by beurotta** [comment]
Duplicate of #755

**Comment by sandrotosi** [comment]
given #755 is completed, im closing this issue as well
